ultraplot/internals/rcsetup.py

```diff
@@ -1575,6 +1575,11 @@ def copy(self):
         "Color for negative bars and shaded areas when using ``negpos=True``. "
         "See also :rcraw:`poscolor`.",
     ),
+    "pcolormesh.grid": (
```

Comment:
There are several ways to plot grids of data, including pcolor and imshow. Do we need to generalize this option to cover all of them? IDK what the semantics of ultrplot is around these redundancies.

Reply:
Good question. Not sure. There is `axes.grid` being set to `True` by default. I can also see that there is a `colorbar.grid` but nothing else. So I assume the rationale is: by default on, only in special cases off.

Reply:
In terms of the other functions would need to check but I am already over my allotted time for this -- so will have to check back in in a few days.